types/src/front/assistant/agent.ts

```diff
@@ -132,3 +132,8 @@ export type AgentConfigurationType = {
   // If undefined, no text generation.
   generation: AgentGenerationConfigurationType | null;
 };
+
+export type AgentUsageType = {
```

Comment:
Should we add a timeperiod information here even if this is monthly for now always?